docs/docs/code/nodejs/browser-automation/README.md

```diff
@@ -0,0 +1,523 @@
+# Browser Automation with Node.js
+
+<VideoPlayer src="https://www.youtube.com/embed/RXlCDTQc8xI" title="Browser automation with Node.js in Pipedream" />
+
+You can leverage headless browser automations within Pipedream workflows for web scraping, generating screenshots, or programmatically interacting with websites - even those that make heavy usage of frontend Javascript.
+
+Pipedream manages a [specialized package](https://www.npmjs.com/package/@pipedream/browsers) that includes Puppeteer and Playwright bundled with a specialized Chromium instance that's compatible with Pipedream's Node.js Execution Environment.
+
+All that's required is importing the [`@pipedream/browsers`](https://www.npmjs.com/package/@pipedream/browsers) package into your Node.js code step and launch a browser. Pipedream will start Chromium and launch a Puppeteer or Playwright Browser instance for you.
+
+[[toc]]
+
+## Usage
+
+The `@pipedream/browsers` package exports two modules: `puppeteer` & `playwright`. Both modules share the same interface:
+
+* `browser(opts?)` - method to instantiate a new browser (returns a browser instance)
+* `launch(opts?)` - an alias to browser()
+* `newPage()` - creates a new page instance and returns both the page & browser
+
+### Puppeteer
+
+First import the `puppeteer` module from `@pipedream/browsers` and use `browser()` or `launch()` method to instantiate a browser.
+
+Then using this browser you can open new [Pages](https://pptr.dev/api/puppeteer.page), which have individual controls to open URLs:
+
+```javascript
+ import { puppeteer } from '@pipedream/browsers';
+
+export default defineComponent({
+  async run({steps, $}) {
+    const browser = await puppeteer.browser();
+    
+    // Interact with the web page programmatically
+    // See Puppeeter's Page documentation for available methods:
+    // https://pptr.dev/api/puppeteer.page
+    const page = await browser.newPage();
+
+    await page.goto('https://pipedream.com/');
+    const title = await page.title();
+    const content = await page.content();
+
+    $.export('title', title);
+    $.export('content', content);
+
+    // The browser needs to be closed, otherwise the step will hang
+    await browser.close();
+  },
+})
+```
+
+#### Screenshot a webpage
+
+Puppeteer can take a full screenshot of a webpage rendered with Chromium. For full options [see the Puppeteer Screenshot method documentation.](https://pptr.dev/api/puppeteer.page.screenshot)
+
+:::: tabs :options="{ useUrlFragment: false }"
```

Comment:
These tabs aren't rendering:

<img width="754" alt="Screenshot 2023-10-12 at 4 51 39 PM" src="https://github.com/PipedreamHQ/pipedream/assets/242668/683dee4a-9202-4ad2-a0cd-3041b72e534b">

Reply:
That is really weird, I wonder if it's some kind of browser incompatibility issue. I can see the tabs just fine:

<img width="797" alt="CleanShot 2023-10-13 at 09 56 18@2x" src="https://github.com/PipedreamHQ/pipedream/assets/2694734/78a355a1-8fba-49c6-ae01-3994c1f5f723">

But I'm going to add a short sentence inside of each tab to compensate for the possibility of this happening on other browsers.